### PR TITLE
Clarify new-app messages

### LINKED ISSUE #10462: new-app template message is misleading
Working in the 'logging' project trying to deploy aggregated logging, deploying the template provides a misleading message.

https://gist.github.com/jcantrill/56d9d4d3225297c4ad71c4034e6b9883

I assume its trying to tell me the project in which the resources are being created but it appears its desplaying the project where the template lives.

##### Version
$ oc version
oc v1.3.0-alpha.3+97d149e-dirty
kubernetes v1.3.0+507d3a7
features: Basic-Auth

Server https://localhost:8443
openshift v1.3.0-alpha.3
kubernetes v1.3.0+507d3a7

##### Steps To Reproduce
1. Add a template to the 'openshift' project
2. Create a new project and change to it.
3. oc new-app TEMPLATE -n openshift

##### Current Result

`Deploying template "logging-deployer-template" in project "openshift"`

##### Expected Result

`Deploying template "logging-deployer-template" in project MY_CURRENT_PROJECT`

### PR DESCRIPTION
**Pull request by jim-minter**
fixes #10462

Suggested message change to:
```
Deploying template "[openshift/]logging-deployer-template" in project MY_CURRENT_PROJECT
```

### CODE REVIEW

File: pkg/generate/app/cmd/describe.go [hunk 1]
```diff
@@ -30,10 +30,10 @@ func displayName(meta kapi.ObjectMeta) string {
 }
 
 func localOrRemoteName(meta kapi.ObjectMeta, namespace string) string {
-	if len(meta.Namespace) == 0 || namespace == meta.Namespace {
-		return meta.Name
+	if len(meta.Namespace) == 0 {
+		return fmt.Sprintf(`"%s"`, meta.Name)
```
Comment (csrwng) [inline pkg/generate/app/cmd/describe.go]: just return the string without the quotes

File: pkg/generate/app/cmd/describe.go [hunk 2]
```diff
@@ -30,10 +30,10 @@ func displayName(meta kapi.ObjectMeta) string {
...
 	}
-	return fmt.Sprintf("%q in project %q", meta.Name, meta.Namespace)
+	return fmt.Sprintf(`"%s/%s"`, meta.Namespace, meta.Name)
```
Comment (csrwng) [inline pkg/generate/app/cmd/describe.go]: same here

File: pkg/generate/app/cmd/template.go [hunk 3]
```diff
@@ -50,9 +50,9 @@ func TransformTemplate(tpl *templateapi.Template, client client.TemplateConfigsN
 func DescribeGeneratedTemplate(out io.Writer, input string, result *templateapi.Template, baseNamespace string) {
 	qualifiedName := localOrRemoteName(result.ObjectMeta, baseNamespace)
 	if len(input) > 0 && result.ObjectMeta.Name != input {
-		fmt.Fprintf(out, "--> Deploying template %s for %q\n", qualifiedName, input)
+		fmt.Fprintf(out, "--> Deploying template %s for %q to project %s\n", qualifiedName, input, baseNamespace)
 	} else {
-		fmt.Fprintf(out, "--> Deploying template %s\n", qualifiedName)
+		fmt.Fprintf(out, "--> Deploying template %s to project %s\n", qualifiedName, baseNamespace)
```
Comment (csrwng) [inline pkg/generate/app/cmd/template.go]: same here

File: pkg/generate/app/cmd/template.go [hunk 4]
```diff
@@ -50,9 +50,9 @@ func TransformTemplate(tpl *templateapi.Template, client client.TemplateConfigsN
 func DescribeGeneratedTemplate(out io.Writer, input string, result *templateapi.Template, baseNamespace string) {
 	qualifiedName := localOrRemoteName(result.ObjectMeta, baseNamespace)
 	if len(input) > 0 && result.ObjectMeta.Name != input {
-		fmt.Fprintf(out, "--> Deploying template %s for %q\n", qualifiedName, input)
+		fmt.Fprintf(out, "--> Deploying template %s for %q to project %s\n", qualifiedName, input, baseNamespace)
```
Comment (csrwng) [inline pkg/generate/app/cmd/template.go]: quote here: 
`Deploying template %q for ...`

**Comment by jim-minter** [comment]
OK, done.

**Comment by csrwng** [comment]
thx, LGTM
[test]

**Comment by jim-minter** [comment]
[test]

**Comment by csrwng** [comment]
[merge]

**Comment by jim-minter** [comment]
rebased and squashed
[test]

**Comment by jim-minter** [comment]
flake #11240
flake #10487

**Comment by csrwng** [comment]
[merge]

**Comment by csrwng** [comment]
flake #11240
flake #10487 
[test]

**Pull request merged**